pol-core/pol/mobile/charactr.cpp

```diff
@@ -3638,12 +3631,14 @@ void Character::unhide()
     if ( client != nullptr )
       send_owncreate( client, this );
 
-    Core::WorldIterator<Core::OnlinePlayerFilter>::InVisualRange(
+    Core::WorldIterator<Core::OnlinePlayerFilter>::InMaxVisualRange(
         this,
         [&]( Character* chr )
         {
           if ( chr == this )
             return;
+          if ( !chr->in_visual_range( this ) )
```

Comment:
just out of curiosity: why isn't in_visual_range() within is_visible_to_me()? optimization?

Reply:
Just checked: for most of the places it would be simpler to add it to. Then there are lines where a new position needs to be checked, ok an additional param with newpos. But then there is something like eg speech this needs to check if visible but with a different range... Gna
So sadly not a simple codecleanup

Reply:
#581